libevm/ExtVMFace.h

```diff
@@ -298,7 +298,7 @@ class ExtVMFace
 	virtual void suicide(Address) { sub.suicides.insert(myAddress); }
 
 	/// Create a new (contract) account.
-	virtual h160 create(u256, u256&, bytesConstRef, OnOpFunc const&) { return h160(); }
+	virtual h160 create(u256, u256&, bytesConstRef, OnOpFunc const&, Instruction) { return h160(); }
```

Comment:
I would better make `OnOpFunc` parameter the last one, because it's kind of different than all the others - all others are configuring the creation and this one is a callback